Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
Moving forward, I believe this will just be `net5.0` (https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-5-preview-3/)